app/javascript/react/components/UserShowTile.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const UserShowTile = (props) => {
+  return (
+    <div>
+      <h1>{props.username}</h1>
+      <h3>{props.zip}</h3>
+      <h3>Reviews by {props.username}:</h3>
+      <ul>
+        <li>Review goes here</li>
+      </ul>
+    </div>
```

Comment:
if your branch doesnt have reviews yet, then you shouldnt make mention of reviews. Delete line 8 through 11. You can add those back in later